src/test/java/LoginPage.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class LoginPage extends BasePage{
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(css ="[type='email']")
+    WebElement emailField;
+    @FindBy(css = "[type='password']")
+    WebElement passwordField;
+    @FindBy(css = "button[type='submit']")
+    WebElement submit;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```

![POM_vs_Factory_1](https://github.com/mlordlie/A49-QA-Automation-Framework/assets/104761482/b3ac4b1d-ddd9-4ebf-9850-6f5f14d39a77)

![POM_vs_Factory_2](https://github.com/mlordlie/A49-QA-Automation-Framework/assets/104761482/e4a9adab-3f58-45f2-a5e2-afae8b4fbfec)